python/paddle/framework/__init__.py

```diff
@@ -54,10 +54,7 @@
 from ..fluid.framework import Parameter
 from ..fluid.dygraph.base import enable_dygraph as disable_static  # noqa: F401
 from ..fluid.dygraph.base import disable_dygraph as enable_static  # noqa: F401
-from ..fluid.framework import _non_static_mode as in_dynamic_mode  # noqa: F401
-from ..fluid.framework import (  # noqa: F401
-    _non_static_mode,  # temporary used for hackson
-)
+from ..fluid.framework import in_dygraph_mode  # noqa: F401
```

Comment:
I suggest changing to `from ..fluid.framework import in_dygraph_mode as in_dynamic_mode`, the reason is in the comments `python/paddle/__init__.py` above

Reply:
done